### PR TITLE
New programmes api

### PR DESCRIPTION
**Pull request by davidrapson**
Adds a new closed programmes listing and updates to use a new version of the programmes content API

### CODE REVIEW

**Comment by davidrapson** [comment]
This will fail until the new API endpoint is live

File: services/content-api.js [hunk 1]
```diff
@@ -9,14 +9,15 @@ const mapAttrs = response => map('attributes')(response.data);
...
+function fetch(urlPath, options = {}) {
     debug(`Fetching ${urlPath}`);
-    const defaults = {
-        url: `${CONTENT_API_URL}${urlPath}`,
-        json: true
-    };
-    const params = Object.assign({}, defaults, options);
-    return request(params);
+    return request({
+        ...{
```
Comment (mattandrews) [inline services/content-api.js]: This syntax will never not look weird to me 😑

File: views/components/programmes.njk [hunk 2]
```diff
@@ -1,35 +1,31 @@
...
-                <dd>{{ fundingRange(
-                    minimum = fields.fundingSize.minimum,
-                    maximum = fields.fundingSize.maximum
-                ) }}</dd>
-            {% endif %}
+
+        {% if programme.fundingSize.description %}
+            <dt>{{ labels.fundingSize }}</dt>
+            <dd>{{ programme.fundingSizeDescription }}</dd>
+        {% elseif programme.fundingSize.minimum and programme.fundingSize.minimum %}
```
Comment (mattandrews) [inline views/components/programmes.njk]: Should one of these be `maximum`?

File: views/components/programmes.njk [hunk 3]
```diff
@@ -1,35 +1,31 @@
...
+            <dt>{{ labels.fundingSize }}</dt>
+            <dd>£{{ programme.fundingSize.minimum | numberWithCommas}}{% if programme.fundingSize.maximum %} to £{{ programme.fundingSize.maximum | numberWithCommas }}{% endif %}</dd>
```
Comment (mattandrews) [inline views/components/programmes.njk]: I think we have the word `to` somewhere in the translations...

**Comment by davidrapson** [comment]
Closing this for now as I pulled most of the new API changes out into a separate PR, will re-raise a clean version of this once we're ready to launch the all programmes list.

**Pull request closed without merging**